File: Java/src/EjerciciosLOO/Ejercicio_09.java
package EjerciciosLOO;

public class Ejercicio_09 {
    public static void main(String[] args) {
        // Sumados: Sumar los 1000 primeros números naturales (1 + 2 + 3 + 4 + … +
        // 1000), imprimiendo por cada suma el resultado parcial obtenido.

        int suma = 0;

        for (int i = 1; i <= 1000; i++) {
            suma += i;
            System.out.println(suma);
        }

    }
}

File: Python/Ejercicios-LOO/ejercicio-09.py
# 9. Sumados: Sumar los 1000 primeros números naturales (1 + 2 + 3 + 4 + … + 1000), imprimiendo por cada suma el resultado parcial obtenido.

sumatory = 0;

for i in range (1, 1001, 1):
    print(f"Suma: {sumatory}");
    sumatory += i;

print(f"La suma total es {sumatory}");